conf/app.ini

```diff
@@ -10,10 +10,6 @@ pipeline:
 [app:h]
 use: call:h.app:create_app
 
-# Elasticsearch configuration
```

Comment:
We shouldn't be defaulting these settings in .ini files. Here we're simply removing the comments. Not sure if there is a reason to keep this in here as a comment. If there is I can put it back.

Reply:
I don't know how we go about documenting required ENVs; this likely isn't the place for them but it's something I'd like to track down at some point.

Reply:
We definitely need this! The refactoring stories will take care of kinda standardizing where that happens in the code and raising an exception if the envvar is required and missing but I do agree it would be good to actually document it somewhere outside of the code for the sanity of everyone.

Reply:
I agree about removing these :+1: I don't think there's a good default Elasticsearch URL in production (everyone's production environment is likely to have a different Elasticsearch URL) so if we had a default Elasticsearch URL in production then the experience of someone setting up h would be that it'd silently use the default URL and then crash with an unhelpful error message when it tried to connect to Elasticsearch.

Instead we should make `ELASTICSEARCH_URL` by a required envvar with no default value in production, so it will crash at config time with a helpful error about how a required envvar is missing.

Also having commented out settings in the ini files seems silly.